pkg/resolution/resource/crd_resource.go

```diff
@@ -116,7 +117,11 @@ func appendOwnerReference(rr *v1beta1.ResolutionRequest, req Request) {
 }
 
 func ownerRefsAreEqual(a, b metav1.OwnerReference) bool {
-	return a.APIVersion == b.APIVersion && a.Kind == b.Kind && a.Name == b.Name && a.UID == b.UID && a.Controller == b.Controller
+	// pointers values cannot be directly compared.
+	if d := cmp.Diff(a.Controller, b.Controller); d != "" {
```

Comment:
Fix a bug.
Previously it was comparing pointers, which always returned false.

Reply:
https://github.com/tektoncd/pipeline/blob/40763d8a3ee63286e265b8fb485af4354bbe6463/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L303-L305

Reply:
I will adjust it.